include/acl_selectors.php

```diff
@@ -625,31 +631,27 @@ function _contact_link($i){ return dbesc($i['link']); }
 				intval($conv_id),
 				dbesc($search),
 				dbesc($search),
-				implode("','", $known_contacts)
+				implode("', '", $known_contacts)
 		);
-		if (dbm::is_result($r)){
+		if (dbm::is_result($r)) {
 			foreach ($r as $row) {
-				// nickname..
-				$up = parse_url($row['author-link']);
-				$nick = explode("/",$up['path']);
-				$nick = $nick[count($nick)-1];
-				$nick .= "@".$up['host'];
-				// /nickname
-				$unknow_contacts[] = array(
-					'type'    => 'c',
-					'photo'   => proxy_url($row['author-avatar'], false, PROXY_SIZE_MICRO),
-					'name'    => htmlentities($row['author-name']),
-					'id'      => '',
-					'network' => 'unknown',
-					'link'    => $row['author-link'],
-					'nick'    => htmlentities($nick),
+				$contact = get_contact_details_by_url($row['author-link'], 0);
```

Comment:
At this point, it's looking for contacts that liked or commented in the thread that we didn't identify before. This means those authors already have been through the probing stage.

Again, I don't want to do any probing in an Ajax call, I'd rather drop the missing contact from the array.